scripts/bash/downloadEnvironmentVariables.sh

```diff
@@ -88,3 +88,10 @@ done
 
 
 bw logout
+
+
+# Clean up detritus on macOS
+# macOS and Ubuntu’s interfaces for sed are slightly different. In this script, we use it in a way
+# that’s compatible to both (by not supplying a suffix for the -i flag), but this causes macOS to
+# generate backup files which we don’t need.
+rm -f ./env/*.env-e ./packages/*/.env-e
```

Comment:
Starting with `./` not strictly necessary, but is safer in case a filename starts with `-` for some reason, which could cause `rm` to try and interpret it as an option/argument